charm_tools/letscharm.py

```diff
@@ -0,0 +1,65 @@
+import subprocess
+
+
+def run_command(command):
+    print(f"Running command {command.split()}")
+    subprocess.call(command, shell=True)
+
+
+def main():
+    print('''
+    Welcome to charming!
+
+    Lets get you started...
+    ''')
+
+    charm_name = input("Whats the charm called?")
+    setup_charm_dir(charm_name)
+    initialise_tree(charm_name)
+
+    k8s_charm = input("Is this a k8s charm? y/n")
+    if k8s_charm.lower() == "y":
+        print("Linking hooks/start")
+        run_command("ln -s ../src/charm.py hooks/start")
+    else:
+        run_command("ln -s ../src/charm.py hooks/install")
+        print("Linking hooks/install")
```

Comment:
These extra outputs seem redundant with the "Running command" output, so I'd suggest removing them both.  However, if you want to keep them, this one should come before the `run_command`